implementing-looker-block.md

```diff
@@ -0,0 +1,7 @@
+# Implementing Shopify's Looker Block
+
+The LookML for this block can be found in this [Github repo](https://github.com/fishtown-analytics/shopify-data-warehouse/tree/readme). 
+
+You can either download the entire block into your Looker application through Github by forking the directory into your repository, copying the SSH URL into a new LookML project, and using the Deploy Key provided by Looker. Detailed directions are outlined [here](https://discourse.looker.com/t/copying-a-lookml-project-into-looker-from-a-github-repo/1209).
+
+However, if you want to selectively migrate pieces of the block, you can simply copy pieces of the block, go to your LookML project, create a new Dashboard file, and paste the chosen LookML code. Directions are outlined [here](https://discourse.looker.com/t/move-a-dashboard-to-a-new-looker-instance/5737). Once you’ve copied the chosen block, skip to step 2
```

Comment:
"skip to step 2" doesn't make sense given in the TOC you're listing this as the final step. Should the order be changed?